### ISSUE TITLE
Strongly Recommend Extending 'eslint:recommended'

### ISSUE THREAD

**Issue by ilyaigpetrov** [google/eslint-config-google#37]
Google eslint rules don't catch typos:
```
function foo(wasModified) {
  bar(wasModifed); // <- Typo!
}
```
So extending `eslint:recommended` is not just a recommendation like "Google is not opinionated blablabla", it is the best practice and strong recommendation.

**Comment by philipwalton** [comment]
The README includes a specific example about using `eslint:recommended` with the Google style rules. I think that's sufficient, and users can decide for themselves whether they want those rules or not.
